projects/ngx-colors/src/lib/components/panel/panel.component.ts

```diff
@@ -213,6 +213,9 @@ export class PanelComponent implements OnInit {
       }
     } else {
       this.format = formats.indexOf(this.service.getFormatByString(this.color));
+      if( this.format < 0 ) {
```

Comment:
why is this validation needed ? getFormatByString should never return a format not found in the array.

Reply:
Because this feature: https://github.com/KroneCorylus/ngx-colors/pull/110

in this feature, we can set the available formats 'hex','rgba','hsla', 'cmyk'

Assuming it is only defined rgb and cmyk, for instance, the line  "this.format = formats.indexOf" will result in a -1 value 

So, the test "if( this.format < 0 )" is for fix it!